here-naksha-lib-hub/src/main/java/com/here/naksha/lib/hub/NakshaHub.java

```diff
@@ -292,28 +296,57 @@ private static WriteXyzCollections createAdminCollectionsRequest() {
 
   @Override
   public @NotNull ExtensionConfig getExtensionConfig() {
+    // Create ReadFeatures Request to read all handlers where extensionId not null from Admin DB
+    final NakshaContext nakshaContext = new NakshaContext().withAppId(NakshaHubConfig.defaultAppName());
+    final ReadFeatures request = new ReadFeatures(EVENT_HANDLERS);
+    final PRef pref = RequestHelper.pRefFromPropPath(EXTN_ID_PROP_PATH);
+    POp notNullCondition = POp.exists(pref);
+    request.setPropertyOp(notNullCondition);
+
+    final Result rdResult;
+    try (IReadSession readSession = getAdminStorage().newReadSession(nakshaContext, false)) {
+      rdResult = readSession.execute(request);
+    } catch (Exception e) {
+      logger.error("Failed during reading extension handler configurations from collections {}. ", request.getCollections(), e);
+      throw new RuntimeException("Failed reading extension handler configurations", e);
+    }
+    final List<EventHandler> eventHandlers;
+    try {
+      eventHandlers = readFeaturesFromResult(rdResult, EventHandler.class);
+    } catch (NoCursor e) {
+      logger.error("NoCursor exception encountered", e);
+      throw new RuntimeException("Failed to open cursor", e);
+    }
+
+    Set<String> extensionIds = new HashSet<>();
+    for (EventHandler eventHandler : eventHandlers) {
+      String extensionId = eventHandler.getExtensionId();
+      if (extensionId != null && extensionId.contains(":")) {
+        extensionIds.add(extensionId);
+      } else {
+        logger.error("Environment is missing for an extension Id");
```

Comment:
also print the value of extensionId which is missing the environment

Reply:
Done.